Implement delete shipment functionality

jtohsohr opened the pull request.

jtohsohr committed:
Implement delete shipment functionality

Bossmanswift commented on .idea/gradle.xml:
```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
This file didn't need to be committed

Bossmanswift committed:
Delete gradle.xml

Avoid modification of Gradle file in the repo at all cost

Bossmanswift committed:
Delete OnShipmentListener.java

Interface not needed at we are not in need of any action when a shipment card is clicked

Bossmanswift committed:
Update ShipmentActivity.java

Bossmanswift committed:
Update ShipmentRecyclerAdapter.java

Bossmanswift committed:
Update ShipmentRecyclerAdapter.java

Bossmanswift committed:
Delete build.gradle

Avoid changing Gradle files

Bossmanswift committed:
Delete gradle-wrapper.properties

Avoid changing properties file
